packages/core/src/Table/stories/TableSection/TableSection.stories.tsx

```diff
@@ -56,5 +68,10 @@ export const TableSettingsStory: StoryObj = {
   parameters: {
     docs: { source: { code: TableSettingsRaw } },
   },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = canvas.getByRole("button", { name: /settings/i });
+    await userEvent.click(button);
```

Comment:
Do we need to add a check to make sure the panel is open when the accessibility tests run?

Reply:
It seems this code runs every time storybook is deployed which means it will also run when the accessibility tests run.